src/components/Proposal/View/ProposalGovernanceSection.tsx

```diff
@@ -50,10 +57,17 @@ function ProposalResultSection({
   const hasVoted = !!(!!userAddress && votes?.[userAddress])
   const showResultsButton = !hasVoted && !finished
 
+  const showPromotionSection = useMemo(
+    () => proposal && proposal.status === ProposalStatus.Passed && PROMOTABLE_PROPOSALS.includes(proposal.type),
+    [proposal]
+  )
+
   useEffect(() => {
     setShowResults(hasVoted || finished || !userAddress)
   }, [hasVoted, finished, userAddress])
 
+  if (!showPromotionSection && !!finished) return null
```

Comment:
Why the promotion section is the condition to render the rest? 🤔

Reply:
Good catch, this got outdated when I removed the other show results section.